MCE.py

```diff
@@ -1531,8 +1531,9 @@ def mass_scan(f_path) :
 		elif cpu_id[2:4] in ['68'] : mc_len = 0x980
 		elif cpu_id[2:4] in ['70','73'] : mc_len = 0xD60
 		elif cpu_id[2:4] in ['80','81','82','83','86','87'] : mc_len = 0xC80
+		elif cpu_id[2:4] in ['A0', 'A2', 'A5'] : mc_len = 0x15C0

```

Comment:
Are we certain that the actual size is 0x15C0? At previous Zen for example, AMD claimed the size to be 0xD00 but was actually 0xC80 with extra (sometimes) 0x80 padding at the end. It is important for the AMD size to be correct/accurate, otherwise we'll end up extracting more or less data.

Reply:
Yes, you can trust me here. I have the BIOS reference code and extracted the microcode from it. For Family 19h the microcode is **always** 0x15C0 in size.